fuzzers/fuzzbench_forkserver_cmplog/src/main.rs

```diff
@@ -355,6 +355,7 @@ fn fuzz(
         let cmpmap = unsafe { OwnedRefMut::from_shmem(&mut cmplog_shmem) };
 
         let cmplog_observer = AFLppCmpLogObserver::new("cmplog", cmpmap, true);
+        let cmplog_ref = cmplog_observer.type_ref();
```

Comment:
Why not `.reference`?

Reply:
Yeah that's more in-line with what we're using now anyways. I'll do a mass find-replace and merge, then.